app/src/display/widgets/battery_status.c

```diff
@@ -64,7 +64,7 @@ void battery_status_update_cb(struct battery_status_state state) {
 
 static struct battery_status_state battery_status_get_state(const zmk_event_t *eh) {
     return (struct battery_status_state) {
-        .level = bt_bas_get_battery_level(),
+        .level = zmk_battery_state_of_charge(),
```

Comment:
It looks like this change in particular was included in https://github.com/zmkfirmware/zmk/commit/0e2f94b73bc18e3be7d913d2a233ae6330b40624#diff-9f9737e080412a2f82ae1867e062e8d581ca4211af5b2a56e64b980668f322d1, which causes a merge conflict. Other instances covered in this PR are not changed, so this is still a helpful PR IMO.

Reply:
Merged, keeping their `ev->state_of_charge`.